index.html

```diff
@@ -0,0 +1,331 @@
+  <dd markdown>The tag for the [version](#versioning) of the spec used to author the document. Processors **must** select an implementation of the spec which can [satisfy](#satisfaction) the specified version.</dd>
+</dl>
+
+The version tag **must** satisfy the [BNF grammar for `‹version tag›`](#backus-naur-form-for-versions-and-version-tags).
+
+### Ignore meaningless URL components
+
+When extracting the URL's `prefix` and `version`, processors **must** ignore any url components which are not assigned a meaning. This spec assigns meaning to the final two components of the [path](https://tools.ietf.org/html/rfc3986#section-3.3). Other URL components—particularly query strings and fragments, if present—**must** be ignored for the purposes of extracting the `prefix` and `version`.
+
+<figure>
+  <figcaption>
+    <a name=spec-url-ignore-meaningless href=#spec-url-ignore-meaningless class=a-header>Ignoring meaningless parts of a URL</a>
+  </figcaption>  
+  <code class=anatomy>
+    <span class=pink style='--depth: 2'>https://example.com/<span>exampleSpec<aside>prefix</aside></span><aside>
+      identity
+    </aside></span>/<span style='--depth: 2' class=green>v1.0<aside>version</aside></span><span class=grey>?key=val&k2=v2#fragment<aside>ignored</aside></span>
+  </code>
+</figure>
+
+### Why is versioning in the URL, not a directive argument?
+
+The version is in the URL because when a human reader visits the URL, we would like them to be taken to the documentation for the *version of the specification in use by this document*. Many text editors will turn URLs into hyperlinks, and it's highly desirable that clicking the link takes the user to the correct version of the docs. Putting the version information in a separate argument to the `@using` directive would require this.
+
+## Prefixing
+Specifications meant to be referenced via `@using` **should** prefix any schema elements they introduce. The prefix **must** be a string of characters valid within GraphQL typenames, with the exception of `"_"`.
+
+Specs **should** define any new schema elements with names that start with `prefix_`.
+
+```graphql
+#::[Using the `@using` directive with a spec's default prefix]
+schema @using(spec: "https://spec.example.com/example/v1") {
+  query: Query
+}
+
+type User {
+  name: String @example_someDirective
+               # prefixed directive from the example spec
+}
+```
+
+The language of the spec **should** specify the default prefix and **should** identify locations where the prefix is used. The prefix **should not** be elided within documentation; definitions of schema elements provided within the spec **must** include the default prefix.
+
+### Elements which should be prefixed
+Specs **should** attach their prefix to:
+- the names of any object types, interfaces, unions, enums, or input object types defined in the spec
+- the names of any fields the spec introduces on *foreign types* defined in a different spec
```

Comment:
(Similar concern to the above.)

Is this intended to be a definition of the term foreign? If so it could be more clear as `introduces on foreign types (i.e., types defined in a different spec)`.

But I also agree with @sachindshinde that specs should be able to introduce special field names that go on non-specced types?  ie you could have a spec that says "you can add a field `caching__policy` to any type and it will be treated as a cache policy" and that's not just on types from specs.